src/components/KTooltip/KTooltip.vue

```diff
@@ -72,63 +71,69 @@ const props = defineProps({
     type: String,
     default: 'auto',
   },
+
   /**
-  * Test mode - for testing only, strips out generated ids
-  */
-  testMode: {
-    type: Boolean,
-    default: false,
+   * @deprecated in favor of text prop
+   */
+  label: {
+    type: String,
+    default: '',
   },
 })
 
 const slots = useSlots()
-const showTooltip = computed((): boolean => !!props.label || !!slots.content)
+const showTooltip = computed((): boolean => !!props.text || !!slots.content || !!props.label)
 
 const computedClass = computed((): string => {
-  let result = ''
-  switch (props.placement) {
-    case 'top':
-      result = 'k-tooltip-top'
-      break
-    case 'right':
-      result = 'k-tooltip-right'
-      break
-    case 'bottom':
-      result = 'k-tooltip-bottom'
-      break
-    case 'left':
-      result = 'k-tooltip-left'
-      break
-  }
+  let placementClass = ''
+  const placementDirections = ['top', 'right', 'bottom', 'left']
+
+  placementDirections.forEach((direction) => {
+    if (props.placement.toLocaleLowerCase().includes(direction)) {
+      if (placementClass) {
+        placementClass += ` tooltip-${direction}`
+      } else {
+        placementClass = `tooltip-${direction}`
+      }
+    }
+  })
 
-  return result
+  return placementClass
 })
 </script>
 
-<style lang="scss">
-.k-tooltip.k-popover {
-  background: var(--kui-color-background-neutral-stronger, $kui-color-background-neutral-stronger);
+<style lang="scss" scoped>
+:deep(.k-tooltip.k-popover) {
+  background-color: var(--kui-color-background-inverse, $kui-color-background-inverse);
   border: none;
+  border-radius: var(--kui-border-radius-20, $kui-border-radius-20);
   color: var(--kui-color-text-inverse, $kui-color-text-inverse);
-  font-size: var(--kui-font-size-30, $kui-font-size-30);
-  padding: var(--kui-space-40, $kui-space-40) var(--kui-space-40, $kui-space-40);
+  font-family: var(--kui-font-family-text, $kui-font-family-text);
+  font-size: var(--kui-font-size-20, $kui-font-size-20);
+  font-weight: var(--kui-font-weight-regular, $kui-font-weight-regular);
```

Comment:
I think we need to bump the font-weight to `500`
```suggestion
  font-weight: var(--kui-font-weight-medium, $kui-font-weight-medium);
```